FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -516,6 +543,48 @@ static bool prvCollectDeviceMetrics( void )
         }
     }
 
+    /* Collect custom metrics. */
```

Comment:
A comment summarizing the custom metrics and their types collected by the demo will be helpful for the reader.